draft-ietf-core-dns-over-coap.md

```diff
@@ -320,42 +320,37 @@ the Content-Format option by the DoC server.
 
 ### Response Codes and Handling DNS and CoAP errors
 
-A DNS response indicates either success or failure in the Response code of
-the DNS header (see {{-dns}} Section 4.1.1). It is RECOMMENDED that
-CoAP responses that carry any valid DNS response use a "2.05 Content"
-response code.
+A DNS response indicates either success or failure in the RCODE of the DNS header (see {{Section 4.1.1 of -dns}}).
+It is RECOMMENDED that CoAP responses that carry a parseable DNS response as defined in {{-dns}} use a "2.05 Content" response code.
```

Comment:
We are not discussing the caching consequences, e.g., setting a good MaxAge.

Reply:
Will add a sentence on that.

Reply:
See 29797726751